packages/sdks/src/blocks/video/video.lite.tsx

```diff
@@ -46,27 +48,75 @@ export default function Video(props: VideoProps) {
     },
     get spreadProps() {
       return {
-        ...props.attributes,
         ...state.videoProps,
       };
     },
   });
   return (
-    <video
-      {...state.spreadProps}
-      preload={props.preload || 'metadata'}
-      style={{
-        width: '100%',
-        height: '100%',
-        ...props.attributes?.style,
-        objectFit: props.fit,
-        objectPosition: props.position,
-        // Hack to get object fit to work as expected and
-        // not have the video overflow
-        borderRadius: 1,
-      }}
-      src={props.video || 'no-src'}
-      poster={props.posterImage}
-    />
+    <div style={{ position: 'relative' }}>
+      <video
+        {...state.spreadProps}
+        preload={props.preload || 'metadata'}
+        style={{
+          width: '100%',
+          height: '100%',
+          ...props.attributes?.style,
+          objectFit: props.fit,
+          objectPosition: props.position,
+          // Hack to get object fit to work as expected and
+          // not have the video overflow
+          zIndex: 2,
+          borderRadius: 1,
+          ...(props.aspectRatio
+            ? {
+                position: 'absolute',
+              }
+            : null),
+        }}
+        src={props.video || 'no-src'}
+        poster={props.posterImage}
+        class="builder-video"
+      >
+        {!props.lazyLoad && <source type="video/mp4" src={props.video} />}
```

Comment:
have we handled case when lazyLoad is true?

Reply:
you're right we are not handling lazyload here